torchmetrics/functional/__init__.py

```diff
@@ -35,6 +35,7 @@
 from torchmetrics.functional.classification.stat_scores import stat_scores
 from torchmetrics.functional.image.gradients import image_gradients
 from torchmetrics.functional.image.psnr import peak_signal_noise_ratio
+from torchmetrics.functional.image.sam import universal_spectral_angle_mapper
```

Comment:
To be coherent with the class metric `SpectralAngleMapper`, it seems that `spectral_angle_mapper` would be a better name.

```suggestion
from torchmetrics.functional.image.sam import spectral_angle_mapper
```

Reply:
I agree. I will revise at your suggestion.